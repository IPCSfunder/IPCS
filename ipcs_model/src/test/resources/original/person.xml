<?xml version='1.0' encoding='UTF-8'?>
<dataset>
	<person_detail></person_detail>
	<school_type SCHOOL_TYPE_OBJID="1" NAME ="HIGH_SCHOOL"/>
	<school SCHOOL_OBJID="1" name = "PUNGOL_PRIMARY_SCHOOL" ADDRESS="PUNGOL" SCHOOL_TYPE_FK="1"/>-
	<role role_objid = "1" name = "ADMIN"/>
	<person person_objid="1" account_name="JamesChen" password_hash="12345" SCHOOL_FK="1"/>
	<person person_objid="2" account_name="WilliamWang"  password_hash="12345" SCHOOL_FK="1"/>
	<person_role PERSON_ROLE_OBJID="1" person_fk="1" role_fk="1"></person_role>
	<relationship_type relationship_type_objid="1" name="PARENT"></relationship_type>
	<contact></contact>
</dataset>