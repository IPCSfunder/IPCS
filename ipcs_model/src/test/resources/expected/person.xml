<?xml version='1.0' encoding='UTF-8'?>
<dataset>
	<person_detail last_name="Gard"  first_Name="Richard" age="25" sex="FEMALE" nationality="Singaporean" dob="1986-08-25" nick_name="GG" nric="1234567" market_option="0"/>
	<school_type SCHOOL_TYPE_OBJID="1" NAME ="HIGH_SCHOOL"/>
	<school SCHOOL_OBJID="1" name = "PUNGOL_PRIMARY_SCHOOL" ADDRESS="PUNGOL" SCHOOL_TYPE_FK="1"/>
	<role role_objid = "1" name = "ADMIN"/>
	<person_role role_fk = "1"></person_role>
	<contact address="Sengkang Street" postcode="12345678" mobile_number="123456" relationship_type_fk="1" primary_contact="1" contact_name="ChildFather"/>
	<relationship_type relationship_type_objid="1" name="PARENT"></relationship_type>
	<person_role role_fk = "1"/>
	<person account_name="JamesChen" password_hash="12345" SCHOOL_FK="1"/>
	<person account_name="WilliamWang"  password_hash="12345" SCHOOL_FK="1"/>
	<person account_name="LuisWang" password_hash="12345" SCHOOL_FK="1"/>
	<activity_type ACTIVITY_TYPE_OBJID="1" NAME ="CLASS"/>
	<activity activity_objid="1" activity_type_fk="1" host="2" name="Math" location="NANNING" description="Math" start_time="2015-05-14 05:00:00" school_fk="1"></activity>
</dataset>